functions-and-operators/json-functions/json-functions-validate.md

```diff
@@ -0,0 +1,377 @@
+1 row in set (0.00 sec)
+```
+
+In the output above you can see that see that validation of the presence of the `fruits`, `vegetables` and `grains` attributes failed as the latter isn't present.
+
+Now let's validate that `fruits` is an array.
+
+```sql
+SELECT JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "array"}}}',@j);
+```
+
+```
++-----------------------------------------------------------------------+
+| JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "array"}}}',@j) |
++-----------------------------------------------------------------------+
+|                                                                     1 |
++-----------------------------------------------------------------------+
+1 row in set (0.01 sec)
+```
+
+The output above confirms that `fruits` is an array.
+
+```sql
+SELECT JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "string"}}}',@j);
+```
+
+```
++------------------------------------------------------------------------+
+| JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "string"}}}',@j) |
++------------------------------------------------------------------------+
+|                                                                      0 |
++------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+The output above tells us that `fruits` is **not** a string.
+
+Now let's also verify the number of items in the array
+
+```sql
+SELECT JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "array", "minItems": 3}}}',@j);
+```
+
+```
++--------------------------------------------------------------------------------------+
+| JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "array", "minItems": 3}}}',@j) |
++--------------------------------------------------------------------------------------+
+|                                                                                    1 |
++--------------------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+The output above tells us that `fruits` is an array with at least 3 items.
+
+```sql
+SELECT JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "array", "minItems": 4}}}',@j);
+```
+
+```
++--------------------------------------------------------------------------------------+
+| JSON_SCHEMA_VALID('{"properties": {"fruits": {"type": "array", "minItems": 4}}}',@j) |
++--------------------------------------------------------------------------------------+
+|                                                                                    0 |
++--------------------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+The output above tells us that `fruits` is **not** an array with at least 4 items. This is because it doesn't meet the minimum number of items.
+
+For integers values we can check if they are in a certain range.
```

Comment:
```suggestion
The preceding output shows that `fruits` is **not** an array with at least 4 items. This is because it does not meet the minimum number of items.

For integers values, you can check if they are in a certain range.
```